code/game/objects/structures/training_machine.dm

```diff
@@ -174,6 +174,11 @@
 
 /obj/structure/training_machine/AltClick(mob/user)
 	. = ..()
+	if(!user.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, NO_TK, FLOOR_OKAY))
+		return
```

Comment:
Should this apply to all altclicks?

Reply:
This is needed to stop cyborgs from incapacitated people etc. from (remotely) unbuckling people when they shouldn't.

Reply:
I'm guessing this fixes some finagling around removing the item in the machines hand too?

Reply:
Yea.